packages/agent/package.json

```diff
@@ -26,10 +25,19 @@
 	"peerDependencies": {
 		"@hyperledger/anoncreds-react-native": "catalog:",
 		"@hyperledger/aries-askar-react-native": "catalog:",
-		"@hyperledger/indy-vdr-react-native": "catalog:",
 		"expo": "catalog:"
 	},
 	"devDependencies": {
 		"@types/fast-text-encoding": "^1.0.3"
+	},
+	"expo": {
```

Comment:
Do we need this in every package?

Reply:
every package that has the expo dependency I added it.

It should only be required for expo + one of the excluded dependencies. But this suffices imo. We can remove it when we move to expo 52.